### ISSUE TITLE
New Quests! [Shigure1408]

### ISSUE THREAD

**Issue by NillerMedDild** [NillerMedDild/Enigmatica2#195]
Merge the new quests into the pack.

**Comment by NillerMedDild** [comment]
Nope, he made new ones, and improved some existing ones. Unfortunately he did so on an outdated version, so he has to re-do them :/

**Comment by BinaryVigilante** [comment]
Oh ok, Thanks!

**Comment by NillerMedDild** [comment]
https://gist.github.com/NillerMedDild/0cb9ca141c1e495c42e19ed7ef83c901
